src/mobility/api/api.ts

```diff
@@ -47,3 +48,22 @@ export const getValueCode = (
       throw error;
     });
 };
+
+const FareProductBenefit = z.object({
+  operator: z.string(),
+  benefits: OperatorBenefitId.array(),
+});
+
+export type FareProductBenefitType = z.infer<typeof FareProductBenefit>;
+
+export const getFareProductBenefits = (
+  _: PreassignedFareProductId,
+): Promise<FareProductBenefitType[]> => {
+  // TODO: Uncomment this when https://github.com/AtB-AS/team-platform/issues/487 is resolved
+  // return client
+  //   .get(`/mobilty/fare-product-benefits/${productId}`, {
+  //     authWithIdToken: true,
+  //   })
+  //   .then((response) => FareProductBenefit.array().parse(response.data ?? []));
```

Comment:
Remember this todo after implementing https://github.com/AtB-AS/team-platform/issues/487